go/cmd/dolt/commands/dump.go

```diff
@@ -39,15 +40,20 @@ import (
 )
 
 const (
-	forceParam = "force"
+	forceParam    = "force"
+	directoryFlag = "directory"
+	filenameFlag  = "file-name"
+
+	sqlFileExt  = "sql"
+	csvFileExt  = "csv"
+	jsonFileExt = "json"
 )
 
 var dumpDocs = cli.CommandDocumentationContent{
 	ShortDesc: `Export all tables.`,
-	LongDesc: `{{.EmphasisLeft}}dolt dump{{.EmphasisRight}} will export {{.LessThan}}table{{.GreaterThan}} to 
-{{.LessThan}}|file{{.GreaterThan}}. If a dump file already exists then the operation will fail,unless the 
-{{.EmphasisLeft}}--force | -f{{.EmphasisRight}} flag is provided. The force flag forces the existing dump file to be 
-overwritten.
+	LongDesc: `{{.EmphasisLeft}}dolt dump{{.EmphasisRight}} will export {{.LessThan}}table{{.GreaterThan}} to {{.LessThan}}file{{.GreaterThan}}. 
```

Comment:
Doc needs updating now that this is getting closer to done (there is no <table> arg, it dumps all tables in the DB)